app/status.go

```diff
@@ -305,19 +305,14 @@ func (a *App) SetStatusDoNotDisturb(userId string) {
 	a.SaveAndBroadcastStatus(status)
 }
 
-func (a *App) SaveAndBroadcastStatus(status *model.Status) *model.AppError {
+func (a *App) SaveAndBroadcastStatus(status *model.Status) {
```

Comment:
We are ignoring error on save status (only logging it). I keep that behavior, but I don't know if want to propagate the error.